pkg/api/org_invite.go

```diff
@@ -132,9 +132,11 @@ func RevokeInvite(c *m.ReqContext) Response {
 	return Success("Invite revoked")
 }
 
+// GetInviteInfoByCode gets a user invite corresponding to a certain code.
+// A response containing an InviteInfo object is returned if the invite is found.
+// If the invite is not found, 404 is returned.
```

Comment:
This comment is a bit misleading; actually, 404 is returned if the invite status is un-expected.

Reply:
@papagian Yeah, it's a question of semantics. I defined it (implicitly) as if a pending invite is not found, since that's all the client should care about (as I defined it).

Reply:
@papagian I reworded the documentation here a bit, to make it clearer.